tools/topology/sof-apl-pcm512x.m4

```diff
@@ -60,37 +59,32 @@ PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s32le.
 # 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
-	2, 1, 2, s32le,
+	2, 5, 2, s32le,
```

Comment:
you need to change the comments as well, it's not 'playback pipeline 2 on PCM 5'. same for all changes below

Reply:
@plbossart thanks for catching, fixed